### DISCUSSION TITLE
new package request: smem

### DISCUSSION THREAD

**Discussion by hualet** [linuxdeepin/developer-center#10416] [Features Request & Ideas | 特性请求 & 头脑风暴]
smem is a useful tool profiling system wide memory consumption.

https://github.com/kwkroeger/smem/tree/master  may not the official repo of this tool , but the author made it python3 compatible and some extra cmdline options like `--cmd-width`,  which is necessary to show the whole process cmdline sometimes.

**Comment by hualet** [comment]
@Zeno-sole

**Reply by Zeno-sole** [reply]
https://github.com/deepin-community/smem/pull/1

**Reply by Zeno-sole** [reply]
https://github.com/linuxdeepin/developer-center/issues/10494  已经到集成流程了  待周五一起推送至内测渠道
